### PR TITLE
FEAT: Adding pysystem-coupling to metapackage

### PR DESCRIPTION
**Pull request by RobPasMue**
As title says.

### CODE REVIEW

**Comment by RobPasMue** [comment]
Missing: image for PySystem coupling. Waiting on inputs from PySystem coupling team

**Comment by PipKat** [comment]
@RobPasMue I don't know why the dev team for System Coupling decided to name the library PySystemCoupling (all as one word), but this is what they used throughout their documentation.

**Review by PipKat — APPROVED** [review]

File: doc/source/examples.rst [hunk 1]
```diff
@@ -81,19 +81,19 @@ PyAnsys packages examples
...
-      :img-top: _static/thumbnails/intro.png
-      :link: https://systemcoupling.docs.pyansys.com/dev/examples/index.html
+    .. grid-item-card:: PyTwin
+      :img-top: _static/thumbnails/pytwin.png
+      :link: https://twin.docs.pyansys.com/dev/examples/index.html
       :text-align: center
       :class-title: pyansys-card-title
 
-    .. grid-item-card:: Granta MI BoM Analytics
+        .. grid-item-card:: Granta MI BoM Analytics
```
Comment (RobPasMue) [inline doc/source/examples.rst]:
```suggestion
    .. grid-item-card:: Granta MI BoM Analytics

```

**Comment by RobPasMue** [comment]
Thanks @PipKat for your contributions! :) Yeah, we'll keep it as they name it. Let's merge it as is - and once we get the image, we will update the repo.

**Pull request merged**